src/modules/Dashboard/components/TakeNowModal/LabeledDropdown/LabeledUserDropdown.tsx

```diff
@@ -69,7 +73,9 @@ export const LabeledUserDropdown = ({
     [debounce, handleSearch],
   );
 
-  const shouldShowWarningMessage = !!canShowWarningMessage && !!value && value.tag !== 'Team';
+  const warningMessageValueCondition =
+    !!value && (enableParticipantMultiInformant ? !value.userId : value.tag !== 'Team');
```

Comment:
Might be good to separate out some of these expressions, as the logic isn't super obvious to me:
```tsx
  const hasAccount = !!value?.userId;
  const isTeamMember = value?.tag === 'Team';
  const warningMessageValueCondition = enableParticipantMultiInformant
    ? !hasAccount
    : !isTeamMember;
```

Reply:
Thanks for the suggestion 👍